### ISSUE TITLE
Hangup During First Run

### ISSUE THREAD

**Issue by pychess** [pychess/pychess#637]
Original [issue 637](https://code.google.com/p/pychess/issues/detail?id=637) reported by [avi.w.levy](https://code.google.com/u/107516950005615666925/) 2011-01-13

**What steps will reproduce the problem?**
1. Ensure that there is no "engines.xml" file
2. Run pychess

**What is the expected output? What do you see instead?**

Program should run properly.
Instead, pychess hangs while configuring engines. See attached logs.

**Does it happen every time?**
Yes

**What version of the product are you using?**
From mercurial source

**Did you use an installed version of PyChess or did you run in from a**
**tarball/svn?**
svn

**Please provide any additional information below.**

**Please attach the latest pychess logfile. It's in a hidden folder, under**
your homedirectory, named ".local/share/pychess/"

**Comment by pychess** [comment]
- attached [2011-01-12_23-01-23.log](https://code.google.com/p/pychess/issues/detail?id=637#c0)
- attached [2011-01-12_23-06-02.log](https://code.google.com/p/pychess/issues/detail?id=637#c0)

**Comment by pychess** [comment]
Original [comment 1.](https://code.google.com/p/pychess/issues/detail?id=637#c1) posted by [avi.w.levy](https://code.google.com/u/107516950005615666925/) 2011-01-13

The gui freezes and shows (in my case) successful install of gnuchess and pychess engines followed by empty throbber for fruit.
I uninstalled fruit just to check and the error still persists; this time gnuchess appears installed and pychess appears uninstalled when the hang occurs.
However, looking through the log output in either case shows that pychess thinks everything installed successfully.

Any idea? It makes testing pychess impossible on this system until I find a workaround.

**Comment by pychess** [comment]
Original [comment 2.](https://code.google.com/p/pychess/issues/detail?id=637#c2) posted by [gbtami](https://code.google.com/u/gbtami/) 2011-01-13

Can't reproduce on Ubuntu 10.04.
What os are you using?
- changed status to: Need-Info
- added label: Defect

**Comment by pychess** [comment]
Original [comment 3.](https://code.google.com/p/pychess/issues/detail?id=637#c3) posted by [avi.w.levy](https://code.google.com/u/107516950005615666925/) 2011-01-14

Ubuntu 10.10

In the mean time, I was able to get past the engine check page and into the main gui. I can try to log in to FICS, and all appears successful until it gets stuck on the "Setting up" something or other message. I can see from the log that I am connected to FICS and able to view everyone's seeks.

So in short, after transitioning from one system to another there have been two unrelated (or so it seems) hang ups in the gui.

**Comment by pychess** [comment]
Original [comment 4.](https://code.google.com/p/pychess/issues/detail?id=637#c4) posted by [lobais](https://code.google.com/u/lobais/) 2011-01-15

Hi Avi,
Can you try cleaning out all pychess related files, and see i that helps?

Thomas

**Comment by pychess** [comment]
Original [comment 5.](https://code.google.com/p/pychess/issues/detail?id=637#c5) posted by [gbtami](https://code.google.com/u/gbtami/) 2013-04-21
- changed status to: WontFix
